dbus-serialbattery/uni_bms.py

```diff
@@ -0,0 +1,447 @@
+############################################################################
+#    Copyright (C) 2024 by macGH                                           #
+#                                                                          #
+#    This lib is free software; you can redistribute it and/or modify      #
+#    it under the terms of the LGPL                                        #
```

Comment:
Version 2 of this software is licensed under the terms of AGPL. Therefore you have to license is also as AGPL, remove this note that the main license applies or stick with the version before v2.

Reply:
Ah forgot it, I refer to the LICENSE file now